### ISSUE TITLE
Creative extra items no longer working

### ISSUE THREAD

**Issue by Packsolite** [maruohon/tweakeroo#395]
The creative extra items feature is no longer working on minecraft 1.19.3, probably due to the new creative tabs.

Minecraft Version: 1.19.3
Mod version: 1.15.0

**Comment by vrad-exe** [comment]
Still an issue on 1.20.1

**Comment by MysticMia** [comment]
1.20.2 also

**Comment by MysticMia** [comment]
It appears they removed the feature in 1.21.1? or hidden it until they work on it again

**Comment by sakura-ryoko** [comment]
The feature has been disabled.  The feature is now a Vanilla Option, called "Operator Tab" since 1.19.X

**Comment by MysticMia** [comment]
I liked being able to put custom items in there though D:

**Comment by vrad-exe** [comment]
Yeah, operator items tab is not a replacement for this. Since it allowed setting custom NBT (now would be item components) I'd use it to store various items with pre configured attributes and enchantments and whatever, so I could retrieve them in any creative world. Or there could be like modded stuff which isn't covered by the vanilla operator items setting...

**Comment by MysticMia** [comment]
A few cases I used this for were light blocks (resolved by Operator Tab) and firework durations like 15-duration for simpler elytra flight. I imagine you can also use it for banners with more than 6 patterns, custom books and signs, slabs/stairs with custom orientations, leather armor with custom colors (instead of having to pry with dyes), etc.
These can all be copied over with saved toolbars too, but being able to edit them in real time can be practical when you're actively building or working with them, as you don't have to switch to a singleplayer or opped world between every use.

**Comment by maruohon** [comment]
When this was first disabled in 1.19.x (1.19.3-ish maybe?), it wasn't "on purpose", but rather due to vanilla code changes making in difficult or not possible to implement without adding an access widener, and I was lazy and didn't feel like doing that change to my dev environment at that time.

It is planned to return at least whenever (if ever?) I start porting my rewrite versions to later MC versions. I don't know if Sakura plans to bring it back to the "old code" versions before that.

**Comment by MysticMia** [comment]
Alright. I hope it does :) Good luck with any rewrites
